pkg/executor/push.go

```diff
@@ -325,3 +381,8 @@ func pushLayerToCache(opts *config.KanikoOptions, cacheKey string, tarPath strin
 	cacheOpts.SkipTLSVerifyRegistries = opts.SkipTLSVerifyRegistries
 	return DoPush(empty, &cacheOpts)
 }
+
+func createECRRepo(svc ecriface.ECRAPI, input *ecr.CreateRepositoryInput) error {
```

Comment:
The only reason this exists is so that sometime in the future, I can add unit tests.